.github/workflows/test.yml

```diff
@@ -21,7 +21,7 @@ jobs:
 
   test:
     name: "Test on Node.js ${{ matrix.node-version }} OS: ${{matrix.os}}"
-    runs-on: ubuntu-latest
+    runs-on: ${{matrix.os}}
     strategy:
       matrix:
         os: [ macos-latest, windows-latest, ubuntu-latest ]
```

Comment:
This CI workflow did not work on Windows…
https://github.com/secretlint/secretlint/tree/master/packages/secretlint/test/snapshots/secretlintignore
https://github.com/secretlint/secretlint/blob/master/packages/secretlint/test/cli.test.ts